storage-transfer/src/main/java/com/google/cloud/storage/storagetransfer/samples/QuickstartSample.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.storage.storagetransfer.samples;
+
+// [START storagetransfer_quickstart]
+import com.google.storagetransfer.v1.proto.StorageTransferServiceClient;
+import com.google.storagetransfer.v1.proto.TransferProto;
+import com.google.storagetransfer.v1.proto.TransferProto.CreateTransferJobRequest;
+import com.google.storagetransfer.v1.proto.TransferTypes.GcsData;
+import com.google.storagetransfer.v1.proto.TransferTypes.TransferJob;
+import com.google.storagetransfer.v1.proto.TransferTypes.TransferSpec;
+
+public class QuickstartSample {
+  /** Quickstart sample using transfer service to transfer from one GCS bucket to another. */
+  public static void quickStartSample(
+      String projectId, String gcsSourceBucket, String gcsSinkBucket) throws Exception {
+    // Your Google Cloud Project ID
+    // String projectId = "your-project-id";
+
+    // The name of the source GCS bucket to transfer objects from
+    // String gcsSourceBucket = "your-source-gcs-source-bucket";
+
+    // The name of the  GCS bucket to transfer  objects to
+    // String gcsSinkBucket = "your-sink-gcs-bucket";
+
+    TransferJob transferJob =
+        TransferJob.newBuilder()
+            .setProjectId(projectId)
+            .setTransferSpec(
+                TransferSpec.newBuilder()
+                    .setGcsDataSource(GcsData.newBuilder().setBucketName(gcsSourceBucket))
+                    .setGcsDataSink(GcsData.newBuilder().setBucketName(gcsSinkBucket)))
+            .setStatus(TransferJob.Status.ENABLED)
+            .build();
+
+    StorageTransferServiceClient storageTransfer = StorageTransferServiceClient.create();
+
+    TransferJob response =
+        storageTransfer.createTransferJob(
+            CreateTransferJobRequest.newBuilder().setTransferJob(transferJob).build());
+
+    storageTransfer
+        .runTransferJobCallable()
```

Comment:
Is this the only way to start the job? Wondering if there's an equivalent method such as `storageTransfer.createTransferJob()` -> `storageTransfer.runTransferJob()`?

Reply:
The only other way would be `runTransferJobAsync()`. There is no `runTransferJob()`.

Reply:
Thanks for checking! Disregard this comment.

Reply:
Would look like

```
storageTransfer.runTransferJobAsync(
            TransferProto.RunTransferJobRequest.newBuilder()
            .setProjectId(projectId)
            .setJobName(response.getName())
            .build())
            .get();
```

Reply:
Well it's nice that we have async, but let's leave that for our readers to find.